workflows-scripts/data-import/SplitFeature_data_import/data-import-file-splitting-feature.jmx

```diff
@@ -0,0 +1,1848 @@
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
+        <elementProp name="ThreadGroup.main_controller" elementType="LoopController" guiclass="LoopControlPanel" testclass="LoopController" testname="Loop Controller" enabled="true">
+          <boolProp name="LoopController.continue_forever">false</boolProp>
+          <stringProp name="LoopController.loops">1</stringProp>
+        </elementProp>
+        <stringProp name="ThreadGroup.num_threads">${DI_VUSERS}</stringProp>
+        <stringProp name="ThreadGroup.ramp_time">${DI_RAMP_UP}</stringProp>
+        <boolProp name="ThreadGroup.scheduler">true</boolProp>
+        <stringProp name="ThreadGroup.duration">${Global_duration}</stringProp>
+        <stringProp name="ThreadGroup.delay">${DI_delay}</stringProp>
+        <boolProp name="ThreadGroup.same_user_on_next_iteration">true</boolProp>
+      </ThreadGroup>
+      <hashTree>
+        <HeaderManager guiclass="HeaderPanel" testclass="HeaderManager" testname="HTTP Header Manager" enabled="true">
+          <collectionProp name="HeaderManager.headers">
+            <elementProp name="" elementType="Header">
+              <stringProp name="Header.name">x-okapi-token</stringProp>
+              <stringProp name="Header.value">${__P(x-okapi-token)}</stringProp>
+            </elementProp>
+          </collectionProp>
+        </HeaderManager>
+        <hashTree/>
+        <JSR223Sampler guiclass="TestBeanGUI" testclass="JSR223Sampler" testname="DI_JSR223 Sampler" enabled="true">
+          <stringProp name="scriptLanguage">groovy</stringProp>
+          <stringProp name="parameters"></stringProp>
+          <stringProp name="filename"></stringProp>
+          <stringProp name="cacheKey">true</stringProp>
+          <stringProp name="script">SampleResult.setIgnore()
+vars.get(&apos;DI_file&apos;).trim().split(&apos;-&apos;).eachWithIndex { DI_file, index -&gt;
+    vars.put(&apos;DI_file_&apos; + ++index, DI_file.trim())
+}</stringProp>
+        </JSR223Sampler>
+        <hashTree/>
+        <ForeachController guiclass="ForeachControlPanel" testclass="ForeachController" testname="ForEach Controller-DI_file" enabled="true">
+          <stringProp name="ForeachController.inputVal">DI_file</stringProp>
+          <stringProp name="ForeachController.returnVal">DI_file</stringProp>
+          <boolProp name="ForeachController.useSeparator">true</boolProp>
+        </ForeachController>
+        <hashTree>
+          <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="DI_Importing MARC records workflow Transaction" enabled="true">
+            <boolProp name="TransactionController.parent">false</boolProp>
+          </TransactionController>
+          <hashTree>
+            <JSR223Sampler guiclass="TestBeanGUI" testclass="JSR223Sampler" testname="DI_JSR223 Sampler- start" enabled="true">
```

Comment:
What is going to happen if I want to run ptf-create-2 profile Data Import?

Reply:
DI_JSR223 Sampler- start should be edited before start
Here DI_profile_name should be changed to ptf-create-2 and 
Prepare files with proper naming. Do not use dashes (-) in file names:
                        bib_Create should be included to the file name for MARC BIB Create
                        holdings_Create should be included to the file name for MARC Holdings Create
                        authority_Create should be included to the file name for MARC Authority Create